Enterprise-CMCS/macpro-mdct-qmr
# Enable ids 2023

### Description
Large update to fix issue in QMR where the label information is being utilized as if it's an id. Any updates made to the label will cause the database identification to change, which creates an issue downstream. 

Only in year 2023 onward, no changes for the previous years.


### Related ticket(s)


MDCT-2268
MDCT-2429

---
### How to test

1) Run QMR and spin up local dynamodb
2) Open rateLabelText.tsx to use as a comparison when viewing the key names in the database.
3) Sign in using account stateuserWA@test.com (this contains AD, CH & HH measures).
4) Select Year 2023
5) This update applies to all measures, so select any one of them
6) For the first 3 questions, select the first radio button. This is will the enable the Performance Measure view
7) Scroll down to Performance Measure and fill out the fields, Numerator & Denominator (Rate will auto populate)
8) Scroll down to Optional Measure Stratification and fill out with any fake data in any of the categories.
9) Click Validate Measure
10) Key fields in local dynamodb for the measure you've just validated should show names that are the same as the IDs field in rateLabelText.tsx

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
